code/modules/antagonists/cult/cult_items.dm

```diff
@@ -506,7 +506,7 @@ Striking a noncultist, however, will tear their flesh."}
 		if(totalcurses == 0)
 			first_curse_time = world.time
 
-		switch(security_num)
+		switch(security_num) //TODO: SET ALERT LEVELS TO CURSE
```

Comment:
just gonna give a small warning, this will most likely conflict with the cult rework pr if testmerged because both touch the same file

Reply:
Gotcha. Im fine waiting for borbop to finish his side of the code first, then I can check it out.